packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-checkbox/gux-form-field-checkbox.scss

```diff
@@ -55,7 +55,7 @@
   display: inline-grid;
   width: var(--gse-ui-checkbox-icon-width);
   height: var(--gse-ui-checkbox-icon-height);
-  margin: 0;
+  margin: 1px 0 0;
```

Comment:
This change seems a bit hacky and does not make a big difference in my opinion. I would remove it.